src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ReferralRepositoryTest.kt

```diff
@@ -224,6 +224,18 @@ class ReferralRepositoryTest @Autowired constructor(
 
   @Nested
   inner class DashboardTypeSelection {
+
+    @Test
+    fun `myCases dashboard will not return self assigned if they're not latest`() {
```

Comment:
What do you think about?
```suggestion
    fun `does not show referrals where the user has been historically assigned but no longer the active assignee`() {
```